### DISCUSSION TITLE
ru/start/join

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#717] [Announcements]
# ru/start/join

Введение Добро пожаловать в наше сообщество! Чтобы пройти проверку на бота и получить роль 

https://korepi.com/ru/start/join.html

**Comment by Qlako** [comment]
Сколько раз пытаюсь, так и не могу зайти в дс :(

**Reply by QweRezOn** [reply]
Пробуй, дискорд открыт

**Reply by Toshimura13** [reply]
Не получается

**Reply by QweRezOn** [reply]
> Не получается

Напиши мне в дискорде-

Qwerezon
